src/ir/branch-utils.h

```diff
@@ -333,6 +352,27 @@ class BranchSeekerCache {
   }
 };
 
+// Provides a map of branch target names to the expression that corresponds to
+// them.
```

Comment:
This is clearer because the previous wording might have described the target block expressions as well.

```suggestion
// Provides a map of branch target names to the expressions that branch to
// them.
```